css/default.css

```diff
@@ -1810,6 +1811,104 @@ input#csds-button {
     .mobile-logo {
         display: flex;
     }
+
+    /* sidebar hamburger effect */
+    #defaultsidebar {
+        height: 100%;
+        position: fixed;
+        left: 0;
+        top: 35px;
+        width: 250px;
+        margin-top: 60px;
+        transform: translateX(-315px);
+        transition: transform 250ms ease-in-out;
+        z-index: 999;
+    }
+
+    .defaultwrapper input[type="checkbox"].openSidebarMenu:checked~#defaultsidebar {
+        transform: translateX(0);
+    }
+
+    input[type=checkbox].openSidebarMenu {
+        transition: all 0.3s;
+        box-sizing: border-box;
+        display: none;
+    }
+
+    .sidebarIconToggle {
+        transition: all 0.3s;
+        box-sizing: border-box;
+        cursor: pointer;
+        height: 100%;
+        width: 100%;
+        height: 22px;
+        width: 22px;
+        margin-bottom: 0;
+        position: fixed;
+        right: 35px;
+        top: 38px;
+        z-index: 999;
+    }
+
+    .spinner {
+        transition: all 0.3s;
+        box-sizing: border-box;
+        position: absolute;
+        height: 3px;
+        width: 100%;
+        background-color: var(--title-text);
+    }
+
+    .horizontal {
```

Comment:
There’s a lot of repetition in these rules (the `transition`, `box-sizing`, other declarations occur several times). Please group the essentials in one rule, and check whether you can even simplify the code (if we don’t use `border-box` on the site, you could still just switch the box model on the parent element, instead of repeating the declaration for every child).